nupic/research/frameworks/dendrites/functional/apply_dendrites.py

```diff
@@ -25,52 +25,67 @@
 a set of dendritic segments.
 """
 
+from collections import namedtuple
+
 import torch
 
+dendrite_output = namedtuple("dendrite_output", ["values", "indices"])
```

Comment:
The docstring is a bit awkward here, but dataclasses don't fit the bill perfectly either. Since they're not tuples, you can't unpack their values (e.g. `valus, indices = dendrite_output`). Note that pytorch return a tuple type from `torch.max`.